README.md

```
@@ -82,13 +88,46 @@ With Novu, you can create custom workflows and define conditions for each channe
-- [Need Help?](https://github.com/novuhq/novu#-need-help)
-- [Links](https://github.com/novuhq/novu#-links)
-- [License](https://github.com/novuhq/novu#%EF%B8%8F-license)
+ 
+- [Why Novu?](#️-why-novu)
+- [Communication infrastructure for products](#-communication-infrastructure-for-products)
+- [Agent Communication Infrastructure (ACI)](#-agent-communication-infrastructure-aci)
+- [Features](#-features)
+- [Getting Started](#-getting-started)
+- [Embeddable Inbox and Preferences](#embeddable-inbox-component)
+- [Providers](#providers)
+- [Need Help?](#need-help)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)